dali/python/nvidia/dali/ops.py

```diff
@@ -352,7 +280,9 @@ def check_args(self):
         self._op.schema.CheckArgs(self._spec)
 
     def generate_outputs(self):
-        pipeline = Pipeline.current()
+        pipeline = _Pipeline.current()
```

Comment:
I wonder if it can be wrapped up into some function which does the check and rises an error? I see that appearing here and there in a couple of places.

Reply:
It used to be like this, but now I made it dependent on the `preserve` attribute.